lte/cloud/go/tools/migrations/m007_na_cleanup/main.go

```diff
@@ -41,7 +41,7 @@ const (
 
 // This migration cleans up cellular gateways on the backend which are
 // "hanging" - i.e. have no associated magmad gateway.
-// See https://github.com/facebookincubator/magma/issues/1071
+// See https://github.com/magma/magma/issues/1071
```

Comment:
Is this really true that the facebookincubator maps 1-to-1 to the Magma repo?
If so, I believe #1071 is closed, i.e. those comments could be removed?